### PR TITLE
Implemented GT, EQ, LT and subtraction

### PR DESCRIPTION
**Pull request by EppeMedia**
Implemented lt, eq, gt between numbers. Implemented difference and subtraction between numbers.

### CODE REVIEW

File: src/numutils.c [hunk 1]
```diff
@@ -3,16 +3,42 @@
...
+
+    return number;
+}
+
 int bstd_number_is_integer(const bstd_number* number) {
     return !number->scale;
 }
 
-int64_t bstd_number_to_int(const bstd_number* number) {
+static int64_t number_to_signed_value(const bstd_number* number) {
```
Comment (omersayilir75) [inline src/numutils.c:27]: agree with the static. We need to make sure it is also done done for other helper functions.

**Review by omersayilir75 — APPROVED** [review]
Conditional: check if the internal functions are all static, then merge.

**Comment by EppeMedia** [comment]
> Conditional: check if the internal functions are all static, then merge.

Shall we merge this now and make a separate issue for those other functions?

**Comment by omersayilir75** [comment]
> > Conditional: check if the internal functions are all static, then merge.
> 
> Shall we merge this now and make a separate issue for those other functions?

sure. 
Made:
https://github.com/Crossover-Compiler/Crossover_bstd_lib/issues/15

**Pull request merged**